### PR TITLE
Use husky to run commands on every commit

### PR DESCRIPTION
**Pull request by RyanGWU82**
Pre-commit: run the linter. 🧹 

Post-commit: rebuild the code in dist/ and add it to the commit. 🤓

("post-commit" seems to be the right hook if you're using `git add`)

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```
Comment (kanadgupta) [inline .husky/pre-commit:4]: This feels a _little_ heavy handed since CI should fail anyways, right?

**Review by kanadgupta — APPROVED** [review]

File: .husky/post-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run build && git add dist
```
Comment (kanadgupta) [inline .husky/post-commit:4]: Also wait sorry should this have a `git commit` command also?

Comment (RyanGWU82) [reply .husky/post-commit:4]: Nope, because it's in the post-commit hook it gets added to the current commit.

Comment (kanadgupta) [reply .husky/post-commit:4]: Ohhhh nice!

File: .husky/pre-commit (hunk 1 above)
Comment (RyanGWU82) [reply .husky/pre-commit:4]: Fair enough, I'll just add this for myself 😆

**Pull request merged**